vue/src/components/Results.vue

```diff
@@ -216,14 +217,53 @@ export default {
 
     forceRerender() {
       this.componentKey += 1;
+      this.results = undefined;
+      this.errors = undefined;
+      
       if (this.single) {
         this.fetchSingleQuery(this.single);
       } else if (this.input) {
         this.fetchQuery(this.input);
       }
     }
   },
-  props: ['input', 'single']
+  props: ['input', 'single'],
+  watch: {
```

Comment:
Ditto here.  We're adding a lot of logic to a single component, and it's unclear if this pattern needs to be applied to all components.  Ideally we would find a cleaner and more generic solution.